tests/parm/input_regional_hafs.nml.IN

```diff
@@ -158,7 +158,12 @@
        fhcyc          = @[FHCYC]
        use_ufo        = .true.
        pre_rad        = .false.
-       imp_physics    = 11
+       imp_physics    = @[IMP_PHYSICS]
+       ltaerosol      = .false.
+       lradar         = .true.
+       ttendlim       = 0.004
+       sedi_semi      = .false.
+       ext_diag_thompson = .false.
```

Comment:
Just to confirm those variables ltaerosol, lradar, ttendlim and sedi_semi are only used by Thompson, they have no impact on other MP schemes, right?

Reply:
Confirmed: those variables ltaerosol, lradar, ttendlim and sedi_semi are only used by Thompson, and they have no impact on other MP schemes.